File: frontend/src/types/scrumboard.ts
export interface ScrumBoard {
    id: number;
    name: string;
    color: string;
    members: number;
    tickets: number;
}

export interface CreateScrumBoardRequest {
    name: string;
    color: string;
    memberIds: string[];
}

export interface ScrumBoardCreatedResponse {
    id: number;
    name: string;
}

export interface ScrumBoardDetail extends ScrumBoard {
    isManager: boolean;
}

File: backend/WorkloadPlanner/Models/ScrumBoard.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace WorkloadPlanner.Models
{
    public class ScrumBoard
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Color { get; set; } = string.Empty;
        public string ManagerId { get; set; } = string.Empty;
        public ApplicationUser? Manager { get; set; }
        public List<BoardMember> Members { get; set; } = [];
        public List<Ticket> Tickets { get; set; } = [];

        [NotMapped]
        public int MemberCount { get; set; }

        [NotMapped]
        public int TicketCount { get; set; }
    }
}